Dealtime/distribution/src/main/assembly/bin.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
License header is not correct. Correct all the places.

Reply:
fixed with - https://github.com/wso2/product-apim-training/pull/2